File: testprojects/csharp/benchmark/Domain/Indirect/Intermediate/ServiceTwo.cs
namespace Domain.Indirect.Intermediate
{
	public class ServiceTwo
	{
		public static string sName = "ServiceTwo";

		public string name;

		public System.DateTime day;

        public Domain.Indirect.IndirectTo.ServiceOne serviceOne;

		public ServiceTwo()
		{
			name = "ServiceTwo";
            day = System.DateTime.Now;
		}

		public virtual string GetName()
		{
			return name;
		}

        public virtual System.DateTime GetDay()
		{
			return day;
		}

		public static string GetsName()
		{
			return sName;
		}

        public Domain.Indirect.IndirectTo.ServiceOne GetServiceOne()
		{
			return serviceOne;
		}
	}
}

File: testresources/java/accuracy/src/domain/indirect/intermediate/ServiceTwo.java
package domain.indirect.intermediate;

import java.util.Calendar;

import domain.indirect.indirectto.ServiceOne;

public class ServiceTwo{
	public static String sName = "ServiceTwo";
	public String name;
	public Calendar day;
	public ServiceOne serviceOne;
	
	public ServiceTwo() {
		name = "ServiceTwo";
		day = Calendar.getInstance();
	}
	public String getName(){
		return name;
	}
	
	public Calendar getDay(){
		return day;
	}

	public static String getsName(){
		return sName;
	}
	
	public ServiceOne getServiceOne(){
		return serviceOne;
	}
	

}